src/Agent/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### New Features
 ### Fixes
+* Fixes issue [#500](https://github.com/newrelic/newrelic-dotnet-agent/issues/500): Agent should always create the "error" intrinsics attribute. ([#501](https://github.com/newrelic/newrelic-dotnet-agent/pull/501))
```

Comment:
It might be worth be a bit clearer on this. How's about ?

"For transactions without errors, Agent still create the `error` intrinsics attribute with its value set to `false`."